tapir/accounts/management/commands/update_purchase_tracking_list.py

```diff
@@ -0,0 +1,64 @@
+import csv
+import os
+
+from django.core.management import BaseCommand
+
+from tapir import settings
+from tapir.accounts.models import TapirUser
+from tapir.settings import GROUP_VORSTAND
+from tapir.utils.user_utils import UserUtils
+
+
+class Command(BaseCommand):
+    help = "Updates the file containing the list of users that allowed purchase tracking and synchronizes it with the BioOffice server."
+    FILE_NAME = "members-current.csv"
+
+    def handle(self, *args, **options):
+        self.write_users_to_file()
+        self.send_file_to_server()
+
+    @classmethod
+    def write_users_to_file(cls):
+        with open(cls.FILE_NAME, "w", newline="") as csvfile:
+            writer = csv.writer(csvfile, delimiter=";", quoting=csv.QUOTE_MINIMAL)
+            writer.writerow(
+                [
+                    "AdressID",  # Must be exactly 12 characters long and start with a 2. Fill with 0
+                    "Nachname",
+                    "Vorname",
+                    "RabattN",
+                    "Strasse",
+                    "PLZ",
+                    "Ort",
+                    "eMail",
+                ]
+            )
+
+            for user in TapirUser.objects.filter(allows_purchase_tracking=True):
+                writer.writerow(
+                    [
+                        "2" + "{:0>11}".format(user.share_owner.id),
+                        user.last_name,
+                        user.first_name,
+                        18 if user.is_in_group(GROUP_VORSTAND) else 0,
+                        UserUtils.get_full_street(user.street, user.street_2),
+                        user.postcode,
+                        user.city,
+                        user.email,
+                    ]
+                )
+
+    @classmethod
+    def send_file_to_server(cls):
+        if not settings.DEBUG:
+            return
+
+        os.system("mkdir ~/.ssh")
```

Comment:
Maybe avoid the key file creation each time the file is uploaded? On the other hand, it is only done once a day.
But then it should be `mkdir -p ~/.ssh` to avoid mkdir throwing an error when the directory already exists.